layers/+completion/spacemacs-ivy/packages.el

```diff
@@ -265,6 +300,26 @@ Helm hack."
         (call-interactively 'describe-mode))
 
       (define-key counsel-find-file-map (kbd "C-h") 'counsel-up-directory)
+
+      ;; set additional ivy actions
+      (ivy-set-actions
+       'counsel-find-file
+       spacemacs-ivy-file-actions)
+
+      (defun spacemacs-ivy/find-file-vsplit (file)
+        "find file in vertical split"
+        (interactive)
+        (select-window (split-window-right))
+        (find-file file)
+        (balance-windows))
```

Comment:
I think we shouldn't call `balance-windows` here, because the user doesn't always want the windows to be balanced (I know I don't).